src/daft-logical-plan/src/ops/window.rs

```diff
@@ -95,7 +96,11 @@ impl Window {
 
 impl Window {
     pub fn multiline_display(&self) -> Vec<String> {
-        let mut lines = vec![format!("Window: {}", self.window_functions.len())];
+        let mut lines = vec![format!("Window: {} functions", self.window_functions.len())];
```

Comment:
No need for the "{} functions" tbh. Especially now since we are displaying all of the functions.